README.md

```diff
@@ -90,6 +90,7 @@ await browserFetcher.DownloadAsync();
 await using var browser = await Puppeteer.LaunchAsync(new LaunchOptions {Headless = true});
 await using var page = await browser.NewPageAsync();
 await page.GoToAsync("http://www.google.com");
+await page.EvaluateExpressionHandleAsync("document.fonts.ready");
```

Comment:
Can we add a comment on top of this line?
```

```suggestion
// You might also need to wait for the fonts to be fully loaded.
await page.EvaluateExpressionHandleAsync("document.fonts.ready");
```

Reply:
Updated. I think the major point here is to outline the possibility of fonts not being loaded and offer some immediate fixes. Might not work in all cases but at least gives a starting point for research.
_(I was thinking for an hour this is due to chromium processess not being disposed of or something due to the bug hitting only on 2nd+ calls)._